diff --git a/src/extractors/sevenzip.rs b/src/extractors/sevenzip.rs
index d8dfd13..2c289e2 100644
--- a/src/extractors/sevenzip.rs
+++ b/src/extractors/sevenzip.rs
@@ -24,7 +24,7 @@ use crate::extractors;
 /// ```
 pub fn sevenzip_extractor() -> extractors::common::Extractor {
     extractors::common::Extractor {
-        utility: extractors::common::ExtractorType::External("7zz".to_string()),
+        utility: extractors::common::ExtractorType::External(if cfg!(windows) { "7z" } else { "7zz" }.to_string()),
         extension: "bin".to_string(),
         arguments: vec![
             "x".to_string(),    // Perform extraction
